dali/python/backend_impl.cc

```diff
@@ -56,15 +56,11 @@ void __backend_impl_force_tls_align_fun(void) {}
 using namespace pybind11::literals; // NOLINT
 
 static void* ctypes_void_ptr(const py::object& object) {
-  PyObject *p_ptr = object.ptr();
-  if (!PyObject_HasAttr(p_ptr, PyUnicode_FromString("value"))) {
+  auto ptr_as_int = getattr(object, "value", py::cast<py::none>(Py_None));
```

Comment:
`py::none()` should be equivalent with `py::cast<py::none>(Py_None)`

Reply:
Done